api/tests/src/legacy_soap_api/grantors/services/test_legacy_soap_api_get_submission_list_expanded_response.py

```diff
@@ -621,23 +716,529 @@ def test_get_submission_list_expanded_response_handles_competition_with_no_oppor
+        sam_gov_entity.has_debt_subject_to_offset = True
+        sam_gov_entity.has_exclusion_status = True
+        user, role, soap_client_certificate = setup_cert_user(
+            agency, {Privilege.LEGACY_AGENCY_VIEWER}
+        )
+        db_session.commit()
+        db_session.refresh(sam_gov_entity)
+
+        setup_application_submission(agency, legacy_competition_id=2)
+        setup_application_submission(agency, legacy_competition_id=3)
+        application = submission.application
+        competition = application.competition
+
+        request_xml = (
+            '<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:agen="http://apply.grants.gov/services/AgencyWebServices-V2.0" xmlns:gran="http://apply.grants.gov/system/GrantsCommonElements-V1.0">'
+            "<soapenv:Header/>"
+            "<soapenv:Body>"
+            "<agen:GetSubmissionListExpandedRequest>"
+            "<gran:ExpandedApplicationFilter>"
+            "<gran:FilterType>CompetitionID</gran:FilterType>"
+            f"<gran:FilterValue>{competition.legacy_competition_id}</gran:FilterValue>"
+            "</gran:ExpandedApplicationFilter>"
+            "</agen:GetSubmissionListExpandedRequest>"
+            "</soapenv:Body>"
+            "</soapenv:Envelope>"
+        )
+        soap_request = SOAPRequest(
+            data=request_xml,
+            full_path="x",
+            headers={},
+            method="POST",
+            api_name=SimplerSoapAPI.GRANTORS,
+            operation_name="GetSubmissionListExpandedRequest",
+            auth=SOAPAuth(certificate=soap_client_certificate),
+        )
+        value = get_soap_operation_dict(request_xml, "GetSubmissionListExpandedRequest")
+        get_submission_list_expanded_rquest_schema = schemas.GetSubmissionListExpandedRequest(
+            **value
+        )
+        get_submission_list_expanded_response(
+            db_session, get_submission_list_expanded_rquest_schema, soap_request
+        )
+        assert (
+            f"GetSubmissionListExpanded Filter: CompetitionIDs ['{competition.legacy_competition_id}']"
+            in caplog.messages
+        )
+
+    def test_get_submission_list_expanded_response_logs_filters_by_package_id(
+        self, db_session, enable_factory_create, caplog
+    ):
+        caplog.set_level(logging.INFO)
+        agency = AgencyFactory.create()
+        sam_gov_entity_1 = SamGovEntityFactory.create(
+            has_debt_subject_to_offset=False, has_exclusion_status=False
+        )
+        submission = setup_application_submission(
+            agency,
+            legacy_package_id="PKG00118065",
+            sam_gov_entity=sam_gov_entity_1,
+            application_status=ApplicationStatus.SUBMITTED,
+        )
+        sam_gov_entity = submission.application.organization.sam_gov_entity
+        sam_gov_entity.has_debt_subject_to_offset = True
+        sam_gov_entity.has_exclusion_status = True
```

Comment:
I'm curious about how this code is setup. You use the factory to create a Sam gov entity. Pass it into the setup function, then pull it out of the submission created and then change the entity?

Why not just set those parameters first? Actually, do these parameters get used by anything you implemented, I don't think we currently use those anywhere, why are setting these at all (before or after)?

Reply:
It's leftover code from other tests that got mixed up. I've tried to go through and clean it up.

These fields are impacted by the sam gov entity:
```
<UEI>
<DelinquentFederalDebt>
<ActiveExclusions>
```